### ISSUE TITLE
Downloading validation set

### ISSUE THREAD

**Issue by ahaliassos** [yinanhe/ForgeryNet#11]
It appears that the validation (and training) set is now stored in a different location. I'm trying to download it from https://www.aliyundrive.com/s/M1FWYLZs4HJ/folder/611f0735642e1e9cf16c4e5d8d74305891be45fb, but because the site is not in English and requires an account, it was not possible to download it.

Would it be possible to move at least the validation set to somewhere else, e.g., GoogleDrive?

**Comment by yinanhe** [comment]
Sorry for this issue. We are in the process of migrating, and the link of Google Drive will be online soon.

**Comment by yinanhe** [comment]
Training set has been shared in https://drive.google.com/drive/folders/15NzY6aVanxg9lTT-zOlG1kqFwbl1FZKr?usp=sharing

**Comment by yinanhe** [comment]
Validation set has been shared in https://drive.google.com/file/d/1RgVa9n9yJDsMZvO6cAsKJH4YtNP9VeNW/view?usp=sharing

**Comment by tech4ap** [comment]
@yinanhe looks like above link to download training and validation dataset is not working. Pls share the updated link.
